lib/pages/WellnessTools/player.dart

```diff
@@ -15,6 +17,8 @@ class VideoPlayerPage extends StatefulWidget {
 class _VideoPlayerPageState extends State<VideoPlayerPage> {
   late VoidCallback listener;
   late YoutubePlayerController controller;
+  bool? _isPlaying;
```

Comment:
why do you need this variable?
can't you pass `controller.value.isPlaying` to the `_logEvent()`?